pom.xml

```diff
@@ -1227,6 +1232,11 @@
         <artifactId>grpc-stub</artifactId>
         <version>${grpc.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>proto-google-common-protos</artifactId>
+        <version>2.9.6</version>
```

Comment:
Similar to other dependencies, can we define versions as variables? (this one and the following 2 versions)

Reply:
+1 on this feedback. @gortiz  Let's define the version as variables

Reply:
I don't understand the advantage of adding variables for these kind of things. That variable should not be read anywhere else, so there is no DRY. It just make the code more difficult to read given you have to move _one thousand_ (!!!) lines above/below to actually know what the variable does.

It would make sense if other projects would use that variable, but that is not the case given we use the value to define a dependencymanagement, whose objective is precisely that subprojects do not write the version by their own.

Anyway, it is not the point of this PR, so I'm changing it.